deb-repo-config.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+OUT="$PWD/repo"
+# KEY=""
+
+_init # Init the repo
+
+for distro in xenial bionic cosmic; do
+  add_dist "$distro" "Orion-$distro" "Orion Debian Packages"
+  add_comp "$distro" stable
+  add_comp "$distro" beta
+  add_arch "$distro" amd64
+  add_arch "$distro" i386
```

Comment:
The builds are just for `amd64` for now, does this affect anything?

Reply:
The i386 will simply be empty. I recommend leaving them in otherwise `apt update` shows some warnings.